### DISCUSSION TITLE
Debugging errors

### DISCUSSION THREAD

**Discussion by borasemiz** [LazyVim/LazyVim#2479] [Q&A] [answered]
I am a very newbie in both vim and neovim. I am setting up an editor configuration for myseld using lazyvim and out of the blue, I get this error on every startup in the screenshot. Sometimes this error is shown on top right corner and disappears very quickly. I checked all my plugins in the `plugins` folder and they look correct. How can I see what the real error message is and which plugin causing it? Also, how can I see them again after they disappear?

```
Invalid plugin spec {
  config = function <1>
}
```

![image](https://github.com/LazyVim/LazyVim/assets/10868750/b9bbe57d-892c-4ec6-bf6d-8a0574b124a7)

**Comment by kobzar** [comment]
Move all your plugins to another dir and try to rerun nvim - if error will gone back plugins by one and you find the file where you take this error.

**Reply by borasemiz** [reply]
The error message dissappears very quickly as well. Is there any log file for these errors where I can see them after disappearing?

**Reply by dpetka2001** [reply, marked as answer]
Type `:messages` or `:Noice all`. These messages persist only during the current session of Neovim.
